### PR TITLE
Use a specitic logger

### PR DESCRIPTION
**Pull request by rgazelot**

### CODE REVIEW

File: src/Resources/config/commands.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
     <services>
         <service id="wisembly.amqp.command.consumer" class="Wisembly\AmqpBundle\Command\ConsumerCommand">
+            <tag name="monolog.logger" channel="amqp" />
```
Comment (rgazelot) [inline src/Resources/config/commands.xml]: The name of the channel is open.

**Comment by Taluu** [comment]
Not sure this is the way to go, as this kind of binds to monolog with a specific logger name which can't be configured.

IMO we should have a compiler pass (or something along those lines) with a configurable key.

Until then, you could create your own command service and erase this service (or better : extend this service)

**Comment by rgazelot** [comment]
That is exactly what is done into the Symfony bundles like [the security](https://github.com/symfony/security-bundle/search?utf8=%E2%9C%93&q=monolog&type=) one.

**Comment by rgazelot** [comment]
And you told me to do that lol. I want to kill you now, you know that ? :p

**Comment by guillaumepotier** [comment]
Use a _specitic_ logger? :trollface:

**Comment by Taluu** [comment]
We could say it's a trademark from @rgazelot. :D

**Pull request merged**